scripts/test.js

```
@@ -0,0 +1,28 @@
+'use strict'
+
+require('perish')
```

just for my education, what does perish do?

---

forces the app to crash when there's an [unhandledrejection](https://developer.mozilla.org/en-US/docs/Web/Events/unhandledrejection) event.  otherwise, promise chains that fail fail silently.  in ~~node 9?, this will be the default behavior.